test/test_integration_variable.py

```diff
@@ -0,0 +1,227 @@
+# -*- coding: utf-8 -*-
+
+"""
+CLI Variable Integration tests
+This test file handles the main test flows for the variable command
+"""
+import io
+import json
+import os
+import tempfile
+import uuid
+from contextlib import redirect_stderr
+from unittest.mock import patch
+
+import Utils
+from conjur.constants import DEFAULT_NETRC_FILE
+from .test_integration_cli import CliIntegrationTest
```

Comment:
Dont import with . at start

Reply:
👌  why don't we like relative imports?

Reply:
Just readbility thing and convention . I think that the most readble is having in the imports full irarchy of folders until the file.

Reply:
Maybe this can answer
https://softwareengineering.stackexchange.com/questions/159503/whats-wrong-with-relative-imports-in-python

Reply:
addressed